packages/apollo-federation/src/service/extractFederationResolvers.ts

```diff
@@ -0,0 +1,47 @@
+import { GraphQLSchema, isObjectType } from 'graphql';
+import { GraphQLResolverMap } from 'apollo-graphql';
+import {
+  GraphQLReferenceResolver,
+  ResolvableGraphQLObjectType,
+} from '../types';
+
+function extractFederationResolversForType(
```

Comment:
Why is `resolvers` plural here? There should only be one `resolveReference` per type.